demo/src/index.js

```diff
@@ -1,14 +1,12 @@
-
 import mirador from 'mirador/dist/es/src/index';
 import annotationPlugins from '../../src';
 import LocalStorageAdapter from '../../src/LocalStorageAdapter';
-import AnnototAdapter from '../../src/AnnototAdapter';
+import RerumAdapter from '../../src/RerumAdapter';
```

Comment:
maybe have this commented out or at least don't replace the AnnototAdapter

Reply:
Ok ok, I wasn't sure how completely we were removing the AnnototAdapter